ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1188,11 +1189,16 @@ class PartCentredForceHolder {
             plugin_.ReportGroundCollision(
                 closest_physical_parent(part1).flightID);
           }
-#if KSP_VERSION_1_9_1
-          foreach (var collider in part1.currentCollisions.Keys) {
-#elif KSP_VERSION_1_7_3
-          foreach (var collider in part1.currentCollisions) {
-#endif
+          var collision_reporter =
+              part1.gameObject.GetComponent<CollisionReporter>();
+          if (part1.gameObject.GetComponent<CollisionReporter>() == null) {
+            // This would only happen if |part1| had been added after
+            // |BetterLateThanNever|, but we never know what the game will throw
+            // at us.
+            continue;
+          }
+          foreach (var collision in collision_reporter.collisions) {
+            var collider = collision.collider;
             if (collider == null) {
```

Comment:
See my comment above and remove this statement.  The comment about #1447 can move to the `CollisionReporter` class.